spec/controllers/checkouts_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+require 'stripe_mock'
+
+RSpec.describe CheckoutsController, type: :controller do
+
+  let(:stripe_helper) { StripeMock.create_test_helper }
+  before { StripeMock.start }
+  after { StripeMock.stop }
+
+  describe "GET#new" do
+    before :each do
+      user = create(:user)
+      sign_in user
+      @shop = create(:shop, :user => user)
+    end
+
+    it "goes to the checkout page" do
+      product = create(:product, shop: @shop)
+      get :new, id: product.id
+      cards = create(:card, :user => subject.current_user)
+      expect(response).to be_success
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe "POST#checkout_current_card" do
```

Comment:
Or maybe something like this? `  expect { Stripe::Charge.create(customer: 'Brian',amount: 1, currency: 'eur') }.to be_present` :P